Backpack/src/main/res/values/dimens.xml

```diff
@@ -10,6 +10,12 @@
   <dimen name="bpk_horizontal_nav_size_normal">36dp</dimen>
   <dimen name="bpk_rating_score_badge_size_small">36dp</dimen>
   <dimen name="bpk_rating_score_badge_size_large">48dp</dimen>
+  <dimen name="bpk_page_title_toolbar_height">48dp</dimen>
```

Comment:
Since we are only using this values in code I think we should stick to using tokens. This one is spacing-lg * 2 for example.